components/viaduct/Cargo.toml

```diff
@@ -11,10 +11,12 @@ crate-type = ["lib"]
 
 [dependencies]
 url = "2.1" # mozilla-central can't yet take 2.2 (see bug 1734538)
+lazy_static = "1.4"
```

Comment:
`viaduct` now uses both `lazy_static` and `once_cell`.
As a followup this should be trimmed down to only one (probably `once_cell` as the more "modern" variant and the API that's gonna get into libstd eventually)

Reply:
Ah, this is of course the first usage of `lazy_static` here. Maybe just use `once_cell` then? We can migrate `lazy_static` in `viaduct-reqwest` later.